corehq/apps/sso/static/sso/js/new_identity_provider.js

```diff
@@ -1,16 +1,26 @@
 hqDefine('sso/js/new_identity_provider', [
     'jquery',
+    'knockout',
     'accounting/js/widgets',
+    'hqwebapp/js/initial_page_data',
 ], function (
     $,
-    widgets
+    ko,
+    widgets,
+    initialPageData
 ) {
 
     var identityProviderModel = function () {
         'use strict';
         var self = {};
 
         self.owner = widgets.asyncSelect2Handler('owner', false, 'select2_identity_provider');
+        self.protocol = ko.observable('saml');
```

Comment:
Is `'saml'` the default? Seems strange that that value is hard-coded here.

Reply:
yep! it is the default